### DISCUSSION TITLE
动态页面变宽

### DISCUSSION THREAD

**Discussion by archerzhoujj** [the1812/Bilibili-Evolved#4849] [功能建议 / Ideas]
能不能将动态页面铺满整个屏幕，左边的直播和右边的话题都顶到屏幕边缘，up主头像选择变成双排或者上下翻页

**Comment by the1812** [comment]
除非重写整个页面否则很难实现

**Reply by archerzhoujj** [reply]
脚本“bilibili页面净化大师”有一个扩增中栏宽度，但只能把中间的动态变宽，而且还会覆盖两边的直播和话题，不是很好用。很难实现就算了，现在也挺好的。谢谢你。
